src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,45 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
+
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = []; // need to use static propery instead
+
+Animal.deleteDead = function() {
+  for (let i = 0; i < Animal.alive.length; i++) {
+    if (Animal.alive[i].health === 0) {
+      Animal.alive.splice(i, 1);
+    }
+  }
+};
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health, hidden = false) {
+    super(name, health);
+    this.hidden = hidden;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(obj) {
```

Comment:
Use better naming

Reply:
![image](https://user-images.githubusercontent.com/71725079/219854136-377de447-7581-412d-9b4e-3ced21ceff5b.png)

Reply:
not allowed to change name here